db/migrations/tbd_add_total_deposits.sql

```diff
@@ -0,0 +1,13 @@
+-- +goose Up
+-- +goose StatementBegin
+SELECT 'up SQL query - add total deposits count and amount columns to stats';
+ALTER TABLE validator_stats ADD COLUMN IF NOT EXISTS deposits_total INT;
+ALTER TABLE validator_stats ADD COLUMN IF NOT EXISTS deposits_amount_total BIGINT;
+-- +goose StatementEnd
+
+-- +goose Down
+-- +goose StatementBegin
+SELECT 'down SQL query - remove total deposits count and amount columns from stats';
+ALTER TABLE validator_stats DROP COLUMN IF EXISTS deposits_total;
+ALTER TABLE validator_stats DROP COLUMN IF EXISTS deposits_amount_total;
+-- +goose StatementEnd
```

Comment:
![image](https://github.com/gobitfly/eth2-beaconchain-explorer/assets/109136188/307ea635-fbee-4040-90c1-953d95300459)
I think this means that file should end with newline?

Reply:
We have some migration files that don't end on a newline but since most of them do I added it.